srcpkgs/ansible/template

```diff
@@ -1,23 +1,14 @@
 # Template file for 'ansible'
 pkgname=ansible
-version=2.9.13
-revision=2
+version=2.10.1
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
-depends="${hostmakedepends} python3-cryptography python3-Jinja2 python3-paramiko
- python3-yaml"
+depends="python3 ansible-base"
 checkdepends="python3-pytest"
 short_desc="Simple deployment, configuration management and execution framework"
-maintainer="Michael Aldridge <maldridge@voidlinux.org>"
```

Comment:
Talked with @the-maldridge on IRC, they're in favour of me taking over.

Reply:
:+1: